app/src/main/java/org/xbmc/kore/ui/widgets/VolumeLevelIndicator.java

```diff
@@ -59,10 +61,18 @@ public VolumeLevelIndicator(Context context, AttributeSet attributeSet, int defS
         initializeView(context);
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        unbinder.unbind();
+        onVolumeChangeListener = null;
+        volumeBarTouchTrackerListener = null;
```

Comment:
Just curious, you're setting the listeners to null now to prevent some leak or just to help the GC? 
Did you ran into some issue?

Reply:
Indeed to help the GC. A possible leak might be the BaseMediaActivity holding a reference to the widget and the widget holding a reference to the Activity through the listener. Although I'm not sure the problem exists here, I just added it as it doesn't hurt.